src/tracing/mux.rs

```diff
@@ -116,6 +126,17 @@ impl MuxInspector {
                         continue;
                     }
                 }
+                TraceConfig::FlatCall(_flatcall_config) => {
+                    if let Some(inspector) = &self.tracing {
+                        inspector
+                            .clone()
```

Comment:
I'm curious if we can avoid clone here.

Reply:
we can't really because we need to consume in the parity tracer